### ISSUE TITLE
Buggy BIOS - there is no boot6

### ISSUE THREAD

**Issue by pjgoodall** [m13253/clover-linux-installer#4]
```
For some buggy BIOS
For some buggy BIOS where BiosBlockIO is needed, search install.sh for boot6 and replace with boot7.
```

There is no `boot6` in install.sh

Can you update the documentation please. I'm not sure if this is the problem I have, nothing happens, but unless I can try the alternative...

**Comment by pjgoodall** [comment]
Looks like you set boot7 as the default

**Comment by m13253** [comment]
Thank you for using my program, and thank you for reporting the issue!

I fixed the problem in commit 489513812b7ec5c23a22a431e9a12c0740c8f174.

**Comment by pjgoodall** [comment]
Thank-you for doing all the work. I ended up successfully using the disk image process. Amazing what you can do with Linux. I learned quite a bit. Thanks!
